### ISSUE TITLE
Wrong type for buttonStyle

### ISSUE THREAD

**Issue by sargunv** [vonovak/react-navigation-header-buttons#13]
The `buttonStyle` prop should be `TextStyle`, not `ViewStyle`. 

In [HeaderButtons.js#L101](https://github.com/vonovak/react-navigation-header-buttons/blob/master/src/HeaderButtons.js#L101), you can see it's being used in a `Text` component.

Simply apply this patch:

```patch
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,9 +1,9 @@
 import { Component, ComponentType, ReactElement } from 'react'
-import { ViewStyle } from 'react-native'
+import { TextStyle, ViewStyle } from 'react-native'
 
 interface HeaderItemProps {
   IconElement?: ReactElement<any>
-  buttonStyle?: ViewStyle
+  buttonStyle?: TextStyle
   buttonWrapperStyle?: ViewStyle
   color?: string
   iconName?: string

```

**Comment by vonovak** [comment]
hi, thanks for reporting. Truth it, it can be either of the two, as seen [here](https://github.com/vonovak/react-navigation-header-buttons/blob/4c714b29c5819c65241809d6abf83d4b69c31aae/src/HeaderButtons.js#L98) where it is used in a View-backed component. I'll update the typings.

**Comment by sargunv** [comment]
Thanks for the quick response! Looks like `TextStyle` extends `ViewStyle`, so it should be good on that front.

**Comment by vonovak** [comment]
@sargunv so do you think this should be kept as-is or changed to `buttonStyle?: TextStyle | ViewStyle`?

**Comment by sargunv** [comment]
Whatever you prefer; `TextStyle` and `TextStyle | ViewStyle` should be effectively identical.

**Comment by vonovak** [comment]
fixed in v1.1.0, I went with `TextStyle | ViewStyle` since it explicitly sends the message that `buttonStyle` can be used to style both
